priv/repo/migrations/20200718042121_modify_unique_index_on_packages.exs

```diff
@@ -0,0 +1,65 @@
+defmodule Hexpm.RepoBase.Migrations.ModifyUniqueIndexOnPackages do
+  use Ecto.Migration
+
+  def up do
+    drop_if_exists(index(:packages, [:repository_id_name]))
```

Comment:
Should we keep this index for lookups such as `Repo.get_by(Package, name: "foo")`. Can postgresql use the `lower(name)` index for this?

If it can't use the new index we should also keep the old index for repositories and organizations.

Reply:
👍
Added indexes to cover queries that don't use lower(name)